components/welcome-block/welcome-block.tsx

```diff
@@ -0,0 +1,79 @@
+'use client';
+import { useTranslations } from 'next-intl';
+import { useState } from 'react';
+
+import { Link } from '@/i18n/navigation';
+
+import { Button } from '../ui';
```

Comment:
Кажется, здесь лучше использовать алиас:
```suggestion
import { Button } from '@/component/ui';
```

Reply:
Окей!